Xamarin.PropertyEditing.Mac/Controls/ThicknessEditorControl.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using AppKit;
+using CoreGraphics;
+using Xamarin.PropertyEditing.Drawing;
+using Xamarin.PropertyEditing.Mac.Resources;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal abstract class ThicknessEditorControl<T> : BaseRectangleEditorControl<T>
```

Comment:
There's only one thickness type, `CommonThickness`. We don't need any generics here.

Reply:
That's across all platforms? Should we remove the generics for Point, Rectangle and Size too, down the line?